src/helper.ts

```diff
@@ -0,0 +1,237 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as gax from 'google-gax';
+import {Descriptors, ClientOptions} from 'google-gax';
+import * as path from 'path';
+import * as gapicConfig from './iam_policy_service_client_config.json';
+import {ProjectIdCallback} from 'google-auth-library';
+import * as protosTypes from '../protos/protos';
+const version = require('../../package.json').version;
+/**
+ *  Google Cloud IAM Client.
+ *  This is manually written for providing methods [setIamPolicy, getIamPolicy, testIamPerssion] to the KMS client.
+ *  We don't support it now in micro-generators.
```

Comment:
Can I have a link to an issue in the `gapic-generator-typescript` repo?  I want to know we're tracking this :)

Reply:
yes, created an issue in `typescript-microgenerator` and link it here.